### PR TITLE
Added alerts based on age of oldest message, with six hour default val

### PR DESCRIPTION
**Pull request by mkvalor-AL**
Note that the default values get set in the lower-level 'module' variables.tf file.  I know some of the description texts don't match perfectly, that's not very important to me.  However, the variable names must match between the respective variable.tf and main.tf names.  There is a lot of repetition so please do your best to check whether you see any mis-matched var names.

### CODE REVIEW

**Comment by mkvalor-AL** [comment]
Just noticed the failing unit test(s).  I'll take a look at it.

**Comment by mkvalor-AL** [comment]
Haha Ok. Not a failed travis test -- simply requires someone to review.

**Comment by automatic-glen-jarvis** [comment]
@mkvalor-AL The tests didn't fail. It is red because a review hasn't been approved.

HOWEVER, there is a test that didn't finish (started but not finished): continuous-integration/travis-ci

File: main.tf [hunk 1]
```diff
@@ -24,8 +27,11 @@ module "queue_high_priority" {
   tags              = "${var.tags}"
 
   queue_alarm_high_message_count_threshold = "${var.queue_alarm_high_priority_high_message_count_threshold}"
-  queue_ok_high_message_count_actions      = "${var.queue_ok_high_message_count_actions}"
   queue_alarm_high_message_count_actions   = "${var.queue_alarm_high_message_count_actions}"
+  queue_ok_high_message_count_actions      = "${var.queue_ok_high_message_count_actions}"
+  queue_alarm_age_limit_exceeded_threshold = "${var.queue_alarm_high_priority_age_limit_exceeded_threshold}"
```
Comment (automatic-glen-jarvis) [inline main.tf]: @mkvalor-AL I don't know context well enough to understand this. But the 

`queue_alarm_age_limit_exceeded_threshold` doesn't match the `queue_alarm_high_priority_age_limit_exceeded_threshold`.

Is this intentional?

File: main.tf [hunk 2]
```diff
@@ -56,8 +65,11 @@ module "queue_bulk" {
   tags              = "${var.tags}"
 
   queue_alarm_high_message_count_threshold = "${var.queue_alarm_bulk_high_message_count_threshold}"
-  queue_ok_high_message_count_actions      = "${var.queue_ok_high_message_count_actions}"
   queue_alarm_high_message_count_actions   = "${var.queue_alarm_high_message_count_actions}"
+  queue_ok_high_message_count_actions      = "${var.queue_ok_high_message_count_actions}"
+  queue_alarm_age_limit_exceeded_threshold = "${var.queue_alarm_bulk_age_limit_exceeded_threshold}"
```
Comment (automatic-glen-jarvis) [inline main.tf]: The `queue_alarm_age_limit_exceeded_threshold` on the left doesn't match the `queue_alarm_bulk_age_limit_exceeded_threshold`. Is that intentional?

File: main.tf [hunk 3]
```diff
@@ -42,6 +48,9 @@ module "queue_low_priority" {
   queue_alarm_high_message_count_threshold = "${var.queue_alarm_low_priority_high_message_count_threshold}"
   queue_ok_high_message_count_actions      = "${var.queue_ok_high_message_count_actions}"
   queue_alarm_high_message_count_actions   = "${var.queue_alarm_high_message_count_actions}"
+  queue_alarm_age_limit_exceeded_threshold = "${var.queue_alarm_low_priority_age_limit_exceeded_threshold}"
```
Comment (automatic-glen-jarvis) [inline main.tf]: The `queue_alarm_age_limit_exceeded_threshold` on the left doesn't match the `queue_alarm_low_priority_age_limit_exceeded_threshold` on the right. Is that intentional?

Comment (automatic-glen-jarvis) [reply main.tf]: Ah, are you just saying "The queue alarm age for this case is the lowest priority limit?"

File: main.tf (hunk 1 above)
Comment (automatic-glen-jarvis) [reply main.tf]: I understand now. This is assigning variables from a pool of preset variables that make sense in the context.

File: main.tf (hunk 2 above)
Comment (automatic-glen-jarvis) [reply main.tf]: I understand now. This is assigning variables from a pool of preset variables that make sense in the context.

File: main.tf (hunk 1 above)
Comment (mkvalor-AL) [reply main.tf]: Yes -- that threw me for a loop at first also. It was the same pattern they used for the high_message_count alert.  Honestly, this should be more granular, but I am sticking with the design we inherited for now, so as not to disrupt any existing configs.

**Pull request merged**